sklearn/metrics/tests/test_classification.py

```diff
@@ -205,10 +205,12 @@ def test_precision_recall_f_binary_single_class():
     assert_equal(1., precision_score([1, 1], [1, 1]))
     assert_equal(1., recall_score([1, 1], [1, 1]))
     assert_equal(1., f1_score([1, 1], [1, 1]))
+    assert_equal(1., fbeta_score([1, 1], [1, 1], 0))
```

Comment:
you need to update the comment above : ``Test precision, recall and F1 score`` -> ``Test precision, recall and fscore`` or something you like.

Reply:
> you need to update the comment above : `Test precision, recall and F1 score` -> `Test precision, recall and fscore` or something you like

Where's this change? You don't need to squash your commits.
Maybe also add ``assert_equal(XXX, fbeta_score([1, 1], [1, 1], float('inf')))`` here.

Reply:
@qinhanmin2014 Looks like there's some accident during I worked on another machine.

Reply:
Maybe also add `assert_equal(XXX, fbeta_score([1, 1], [1, 1], float('inf')))` here.

Reply:
@qinhanmin2014 
https://github.com/scikit-learn/scikit-learn/pull/13231/commits/aa90c289383742f1b69d5d1d061cfa8d6458f603
I've added the test for that case.
Due to https://bugs.python.org/issue28579, I use numpy.isnan method for this test.

Reply:
assert_true is deprecated, use bare assert instead

Reply:
@qinhanmin2014 done